crates/hyperqueue/src/common/cli.rs

```diff
@@ -303,6 +308,25 @@ pub struct HwDetectOpts {
     pub no_hyper_threading: bool,
 }
 
+#[derive(Parser, Debug)]
+pub struct DeploySSHOpts {
```

Comment:
I believe that in Rust naming convention it should `DeploySshOpts` (for example `TcpListener`).